data/toc.yaml

```diff
@@ -1946,7 +1946,7 @@ Manuals:
           - path: /build/ci/github-actions/
             title: Introduction
           - path: /build/ci/github-actions/build-summary/
-            title: Build summary
+            title: Build summary {{< badge color=amber text=Beta >}}
```

Comment:
Maybe `blue` to match the callouts? Not sure how it looks in the toc tho
<img width="334" alt="image" src="https://github.com/docker/docs/assets/35727626/6dc1678e-cf33-45a1-93bd-9465026e9e51">


```suggestion
            title: Build summary {{< badge color=blue text=Beta >}}
```

Reply:
![image](https://github.com/docker/docs/assets/1951866/06a226e9-468e-4eaf-8810-8d53c211b98b)